.gitignore

```diff
@@ -0,0 +1 @@
+flavor
```

Comment:
I'm not really sure. I just placed it anywhere and figured it wouldn't be much of a use since we're getting rid of ZODB